### ISSUE TITLE
Bug: Prerender activity button broken in Firefox

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetTools#1763]
The prerender activity button works fine in Chrome.  However, in Firefox, it gives the error:
```
SyntaxError: import declarations may only appear at top level of a module
```
This is presumably due to Firefox not supporting es6 web workers and we haven't compiled the prerender activity web worker down to cjs.

**Comment by Lyanthropos** [comment]
This thread ([link](https://bugzilla.mozilla.org/show_bug.cgi?id=1247687)), has updates on the subject of firefox support. Promisingly the assigned made changes two hours ago...

**Comment by dqnykamp** [comment]
Yeah, I've been keeping on eye on that progress since we moved core to a web worker last year.  They will eventually roll that out!
